unittests/ConfigParserTests.cpp

```diff
@@ -43,6 +42,66 @@ TEST_F(ConfigParserTestFixture, loadConfig_bitcodeFileList) {
   ASSERT_EQ(errors.size(), 0U);
 }
 
+TEST_F(ConfigParserTestFixture, loadConfig_dynamicLibraryFileList_nonExistingFile) {
+  static std::string BitcodeFileList = "/tmp/bitcode_file_list.txt";
```

Comment:
Also please always use `camelCase` for local variables: `bitcodeFileList`, not `BitcodeFileList`. Historically, @AlexDenisov started mull to follow LLVM conventions that include capitalization of local variables but later we both couldn't stand it and switched to a `lowerCamelCase`. So currently the code has both styles and the preferred one we are moving to is `lowerCamelCase`. Btw the same goes to class instance variables.